proposed-charter.txt

```diff
@@ -166,6 +166,10 @@ Decision Policy
 
 As explained in the Process Document (section 3.3), this group will seek to make decisions when there is consensus. When the Chair puts a question and observes dissent, after due consideration of different opinions, the Chair should record a decision (possibly after a formal vote) and any objections, and move on.
 
+Editors are responsible for reflecting the consensus from the Working Group in the specifications; where editors bring technical solutions in the specifications that have not been reviewed by the group, these solutions are annotated to reflect their status.
+
```

Comment:
I don't see why this should be limited to editor solutions. If the point is that there are provisional changes that don't yet have consensus, they should be so annotated regardless.